spec/requests/onboardings_spec.rb

```diff
@@ -14,5 +14,22 @@
       get onboarding_url
       expect(response).to have_http_status(:ok)
     end
+
+    it "contains proper data attribute keys" do
+      sign_in user
+      get onboarding_url
+      expect(response.body).to include("community-description")
+      expect(response.body).to include("community-logo")
+      expect(response.body).to include("community-background")
+      expect(response.body).to include("data-community-name")
```

Comment:
Should this not just have been "community-name" like the rest?

Reply:
✅